File: A1_q1.c
/*
 * C program to find the sum of 'N' natural numbers
 */
#include <stdio.h>
 
  int main()
{
	//intialise the value
    int i, num, sum = 0;
 
    printf("Enter an integer number \n");
    scanf ("%d", &num );
    if (num<0)
    {
    	printf("Enter the positive integer.");
	}
	else
    {
	
       for (i = 1; i <= num; i++)
        {
        sum = sum + i;
        }
      
    printf ("Sum of first %d natural numbers = %d", num, sum);
    }
}

File: A1_Q1.py
# add 1+2+3...100 WITHOUT using the formula n(n+1)/2

#taking input from user
num = int(input("Enter the number : "))
sum = 0

if num < 0:
   print("Enter a positive number")
else:
   # use while loop to iterate until zero
   while(num > 0):
       sum += num
       num -= 1
print("The sum is", sum ,".")
